packages/vue/src/create-element.js

```diff
@@ -17,45 +14,35 @@ const defaults = {
 
 const own = {}.hasOwnProperty
 
-export default function createMdxElement(type, props, ...children) {
-  let node
-
+export default function createMdxElement(type, props, children) {
```

Comment:
I made it pretty hard for myself last time by thinking it’d be wise to also support the React babel transform, for Vue.
That was stupid. Now our Vue runtime only supports the Vue babel transform.